.changes/dead-code-warning.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
I think the warning is only emitted when developing this crate, so no need to add a change file

Reply:
Thanks. I confirmed that this warning was not reported when it is used as dependency. I will remove it.

Reply:
Removed: fdee0a2659f8f7b5e540e629b6324624657c304c